rebasehelper/logger.py

```diff
@@ -132,65 +182,35 @@ def add_file_handler(logger_object, path, formatter_object=None, level=None):
-            CustomLogger.IMPORTANT: {'fg': 'red', 'bg': 'default', 'style': None},
-            logging.WARNING: {'fg': 'blue', 'bg': 'default', 'style': None},
-            logging.ERROR: {'fg': 'red', 'bg': 'default', 'style': 'bold'},
-            logging.CRITICAL: {'fg': 'white', 'bg': 'red', 'style': 'bold'},
-        },
-    }
+def create_file_handlers(results_dir: str) -> str:
+    """Creates rebase-helper file handlers.
 
-    terminal_background: str = 'dark'
+    Args:
+        results_dir: Path to rebase-helper-results directory.
 
-    def set_terminal_background(self, background):
-        if background == 'auto':
-            self.terminal_background = ConsoleHelper.detect_background()
-        else:
-            self.terminal_background = background
+    Returns:
+        Path to debug log file.
 
-    def emit(self, record):
-        try:
-            message = self.format(record)
-            level_settings = self.colors[self.terminal_background].get(record.levelno, {})
-            ConsoleHelper.cprint(message, **level_settings)
-            self.flush()
-        except Exception:  # pylint: disable=broad-except
-            self.handleError(record)
-
-
-logging.setLoggerClass(CustomLogger)
-#  the main rebase-helper logger
-logger: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper')
-#  logger for output tool
-logger_output: CustomLogger = LoggerHelper.get_basic_logger('output-tool', logging.INFO)
-logger_report: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper-report', logging.INFO)
-logger_traceback: CustomLogger = LoggerHelper.get_basic_logger('traceback', CustomLogger.TRACE)
-logger_upstream: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper-upstream')
-
-console_formatter: logging.Formatter = logging.Formatter("%(levelname)s: %(message)s")
-log_formatter: logging.Formatter = logging.Formatter("%(message)s")
-debug_log_formatter: logging.Formatter = logging.Formatter(
-    "%(asctime)s %(filename)s:%(lineno)s %(funcName)s: %(message)s")
-
-main_handler: ColorizingStreamHandler = LoggerHelper.add_stream_handler(logger, logging.INFO, console_formatter)
-output_tool_handler: ColorizingStreamHandler = LoggerHelper.add_stream_handler(logger_output)
+    """
+    logs_dir = os.path.join(results_dir, constants.LOGS_DIR)
+    logger = logging.getLogger("rebasehelper")
+
+    log_formatter = logging.Formatter("%(message)s")
+    debug_log_formatter = logging.Formatter("%(asctime)s %(filename)s:%(lineno)s %(funcName)s: %(message)s")
+
+    debug_log = os.path.join(logs_dir, constants.DEBUG_LOG)
+    LoggerHelper.add_file_handler(logger, debug_log, debug_log_formatter, logging.DEBUG)
+    verbose_log = os.path.join(logs_dir, constants.VERBOSE_LOG)
+    LoggerHelper.add_file_handler(logger, verbose_log, log_formatter, CustomLogger.VERBOSE)
+    info_log = os.path.join(logs_dir, constants.INFO_LOG)
+    LoggerHelper.add_file_handler(logger, info_log, log_formatter, logging.INFO)
+    return debug_log
```

Comment:
I think returning `debug_log` is unnecessary. `Appplication` shouldn't need to access it at all.